src/ophyd_async/fastcs/panda/__init__.py

```diff
@@ -28,25 +31,27 @@
 
 __all__ = [
     "CommonPandaBlocks",
-    "HDFPanda",
-    "PcompBlock",
-    "PcompInfo",
-    "PcompDirectionOptions",
+    "DataBlock",
```

Comment:
Could: tempted to replace `Block` -> `IO` here. `PcompIO` probably fine, but not sure if I like `DataIO`... Is `DataBlockIO` too much of a mouthful? Or shall we make PandA a special case that has `Block` instead of `IO`?

Reply:
I don't have strong opinions on PandA namings, but perhaps this can be done in a subsequent PR after further discussion?

Reply:
Leave as is for now